internal/ctrlflow/ssa.go

```diff
@@ -0,0 +1,42 @@
+package ctrlflow
+
+import (
+	"go/constant"
+	"go/types"
+	"reflect"
+	"unsafe"
+
+	"golang.org/x/tools/go/ssa"
+)
+
+// setUnexportedField is used to modify unexported fields of ssa api structures.
```

Comment:
this probably warrants a TODO to raise a feature request upstream, or find some other way to do this that doesn't require unsafe.